### DISCUSSION TITLE
Question regarding building of own kubebench image

### DISCUSSION THREAD

**Discussion by jaanhio** [aquasecurity/kube-bench#958] [Questions and Help]
hi all, i am a new user of kubebench on my EKS cluster

i was reading through the guide and came across the part where we are instructed to build our own kubebench image before pushing to ECR and referencing that image instead. https://github.com/aquasecurity/kube-bench/blob/main/docs/running.md#running-in-an-eks-cluster

question i have is, why though? why cant we run kubebench using `aquasec/kube-bench:latest`?

**Comment by gugu** [comment]
I have a guess it is because of dockerhub pull rate limits, but it is possible now to build one public AWS ECR image instead
